backend/app/api/v1/admin.py

```
@@ -67,27 +66,38 @@ async def upload_report_template(
-    db: AsyncSession = Depends(get_db)):
+    db: AsyncSession = Depends(get_db),
+):
     try:
-        return await admin_service.add_clinical_protocols(db=db, file=file, uploaded_by_user_id=admin.id, docs_path="clinical_protocols")
+        return await admin_service.replace_clinical_protocols(
+            db=db,
+            files=files,
+            uploaded_by_user_id=admin.id,
+        )
     except Exception as e:
         raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e))
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)